locust/env.py

```diff
@@ -66,7 +75,7 @@ class Environment:
     def __init__(
         self,
         *,
-        user_classes=[],
+        user_classes=None,
```

Comment:
It's considered bad practice to set keyword-arguments to mutable type.

There's plenty of reference on that on the internet, but here's one of the first result on Google: https://florimond.dev/en/posts/2018/08/python-mutable-defaults-are-the-source-of-all-evil/.

Reply:
I'm familiar with that gotcha, but in this case I think the current implementation is better (less code, and we know we're never going to modify user_classes anyway). And it doesnt really relate to this PR...

Reply:
Yes it's true that it is out of scope of this PR. I just couldn't resist changing it to be more robust.

I'll revert it, but I still think this should be changed in another PR because it makes the code more robust and for contributors less versed into Python and this gotcha, it might prevent unwanted bugs down the road.

Even my IDE warns me about this.